### ISSUE TITLE
[Build error] @ember/debug does not have a deprecate import

### ISSUE THREAD

**Issue by Fed03** [yapplabs/ember-modal-dialog#203]
Hi, I've just upgraded my project to ember-cli 2.13.3 and suddenly this happens during build 

```
'instrument' is imported from external module 'ember-data/-debug' but never used
The Broccoli Plugin: [BroccoliMergeTrees: Addon#compileAddon(ember-modal-dialog) ] failed with:
SyntaxError: ember-modal-dialog/components/deprecated-tether-dialog.js: @ember/debug does not have a deprecate import
       2 | import BasicDialog from './basic-dialog';
       3 | import layout from '../templates/components/deprecated-tether-dialog';
    >  4 | import { deprecate } from '@ember/debug';
         | ^
       5 |
```

**Comment by mazondo** [comment]
Here's a fix that worked for me: https://github.com/yapplabs/ember-modal-dialog/pull/204

**Comment by Dhaulagiri** [comment]
This seems related to https://github.com/emberjs/data/issues/5016??

**Comment by lukemelia** [comment]
Closing as fixed by #204
